src/sentry/llm/open_ai.py

```diff
@@ -0,0 +1,44 @@
+from sentry.runner import configure
+
+configure()
+from django.conf import settings
+from openai import OpenAI
+
+from sentry.llm.base import LLMBase
+
+
+class OpenAIProvider(LLMBase):
+    def complete_prompt(self, prompt: str, message):
+        model = settings.SENTRY_LLM_OPTIONS["model"]
+        client = get_openai_client()
+
+        response = client.chat.completions.create(
+            model=model,
+            temperature=0.7,
```

Comment:
can we abstract out temperature so its not hardcoded, and instead an input to `complete_prompt`? we'll probably want to normalize -- i believe openAI's temperature is from 0 to 2, and Vertex is between 0 and 1.

Reply:
should we pass in `temperature` & `candidate_count` & `top_p` etc in `vertex.py` as well? currently they're hardcoded on the class

Reply:
no -- we only need to do this for things we imagine we'll want to change. for now, candidate_count, top_p i can't imagine we'd want to change, at least in the short term. 

we also need to keep in mind that we need to keep the parameters model agnostic -- you should be able to swap out vertex for openAI without having to change what the code does.